scripts/docker/dockerfile_clang-10

```diff
@@ -12,9 +12,11 @@ WORKDIR "/home/serac"
 USER serac
 
 RUN git clone --recursive https://github.com/LLNL/serac.git
+RUN cd serac && git checkout feature/white238/addinlet && cd ..
```

Comment:
Should we leave a hard call to this branch name here?

Reply:
These are generated by the build_new_image script.  They probably shouldn't be checked in.

Reply:
This is fixed by #233